pubsub/rabbitmq/rabbitmq_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func createAmqpMessage(body string) amqp.Delivery {
```

Comment:
changes in this file can be exlcluded from PR as its not in the scope of the PR.